themes/godotengine/layouts/home.htm

```diff
@@ -197,98 +247,108 @@ <h3 class="base-padding no-margin-bottom">News</h3>
 <section id="features" class="container padded">
   <h2>A different way to make games</h2>
 
-  <div class="flex eqsize responsive features-row">
-    <a href="/features#design" data-barba-prevent>
-      <div class="feature">
+  <div class="features-grid">
+    <a class="feature-link" href="/features#design" data-barba-prevent>
+      <div class="feature dark">
         <img
-          src="{{ 'assets/home/features/innovative.png' | theme }}"
+          src="{{ 'assets/home/features/innovative.webp' | theme }}"
           alt=""
           width="1"
           height="1"
           style="background: linear-gradient(90deg, #333747 11%, #2d3342 50%, #2d3342 68%, #272d3c 87%)"
           loading="lazy"
         >
-        <div class="dark base-padding">
+        <div class="body">
           <h4>Innovative design</h4>
-          <p>Big or small ideas adapt seamlessly to Godot's node-based architecture, making your life easier.</p>
+          <p>
+            Godot's Node and Scene systems give you both power and flexibility
```

Comment:
Maybe "system" should be singular? Nodes and Scenes are two sides of the same coin, to me it's a single system.